styles.css

```diff
@@ -415,3 +432,243 @@ form {
   padding-top: 1.875rem;
   padding-bottom: 0;
 }
+
+/* Desktop version */
+@media only screen and (min-width: 961px) {
+  body {
+    background: #e5e5e5;
+    width: 100vw;
+  }
+
+  .toggle-menu {
+    display: none;
+  }
+
+  header {
+    background-color: #fff;
+    width: 100%;
+  }
+
+  .navbar {
+    padding-top: 0.8rem;
+    padding-left: 9.688rem;
+    padding-right: 9.688rem;
+    padding-bottom: 0.8rem;
+  }
+
+  .my-logo {
+    width: 280px;
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .nav-list {
+    display: flex;
+    width: 18.5rem;
+    justify-content: space-around;
+    align-items: center;
+    margin: 0;
+    padding: 0;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .nav-list a {
+    color: #344563;
+  }
+
+  .headline {
+    background-color: #fff;
+    background-image: url(./images/desktop/Header-bg-desktop.svg);
+    background-repeat: no-repeat;
+    background-size: 65rem;
+    background-position-y: 15px;
+    border-bottom-left-radius: 100px;
+    height: 90vh;
+    width: 100%;
+    margin: 0;
+  }
+
+  .heading,
+  .paragraph-one {
+    padding-left: 16.25rem;
+    margin-left: 0;
+    padding-right: 16.25rem;
+  }
+
+  .heading-one {
+    padding-top: 10.438rem;
+  }
+
+  .head-paragraph,
+  .header-connect {
+    padding-left: 10rem;
+  }
+
+  .lets-connect a {
+    font-size: 16px;
+  }
+
+  .header-socios {
+    padding-left: 10rem;
+  }
+
+  .social-media-links {
+    width: 8.75rem;
+    margin-left: 5.875rem;
+  }
+
+  .social-media-links img {
+    width: 0.875rem;
+    height: 0.938rem;
+  }
+
+  /* desktop version works section */
+  .grid-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background: #e5e5e5;
+    margin-bottom: 20px;
+  }
+
+  #portfolio {
+    margin-left: 20px;
+  }
+
+  .grid-item {
+    display: flex;
+    width: 62.5rem;
+    height: 25rem;
+    justify-content: space-between;
+    padding: 1.25rem;
+  }
+
+  .card-description {
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+    margin: 10px;
+  }
+
+  .project {
+    width: 50%;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    height: 100%;
+  }
+
+  .second-project {
+    order: 2;
+  }
+
+  .fouth-project {
+    order: 2;
+  }
+
+  .project img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .frame {
+    display: flex;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    justify-content: space-between;
+    width: 220px;
+  }
+
+  .frame ul {
+    display: flex;
+    list-style-type: disc;
+    justify-content: space-between;
+    width: 180px;
+  }
+
+  .frame li {
+    padding-left: 10px;
+  }
+
+  /* About myself section */
+  #about-myself {
+    display: flex;
+    padding: 0;
+    border-top-right-radius: 100px;
+  }
```

Comment:
- Good job on your about section 👍, I think it is better to add padding on the left and right to fit directly by the card direction👌.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206843230-564ef549-35fc-4568-991f-801608dfe47e.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206843231-28d49941-37c5-497f-971b-a2621b1a1ad2.png" width="200">|


😎️

Reply:
Hi @iambenkis, The padding has been reduced to match the design.
Thank you for reviewing my PR.